roadmap/src/gql/queries.js

```diff
@@ -10,6 +10,9 @@ export const GET_ROADMAP = gql`
       _id
       title
       description
+      upvotes {
+        upvotes
```

Comment:
why this nesting? if it denotes number of upvotes name the inner parameter `count`